src/vm/docs/mod.rs

```diff
@@ -241,6 +241,55 @@ Note: This function is only available starting with Stacks 2.1.",
 "#,
 };
 
+const STRING_TO_INT_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: None,
+    signature: "(string-to-int (string-ascii|string-utf8))",
+    description: "Converts a string, either `string-ascii` or `string-utf8`, to a signed integer.
+
+    Note: This function is only available starting with Stacks 2.1.",
+    example: r#"
+(string-to-int "1") ;; Returns 1
+(string-to-int u"-1") ;; Returns -1
+    "#,
+};
+
+const STRING_TO_UINT_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: None,
+    signature: "(string-to-uint (string-ascii|string-utf8))",
+    description:
+        "Converts a string, either `string-ascii` or `string-utf8`, to an unsigned integer.
+
+        Note: This function is only available starting with Stacks 2.1.",
+    example: r#"
+(string-to-uint "1") ;; Returns u1
+(string-to-uint u"1") ;; Returns u1
```

Comment:
What happens if you give it an invalid numeric representation, like `"abc"`?  What's the error code?

Reply:
I changed this so now "string to int" functions will return an "optional", and updated the docs.